packages/location-management/src/components/LocationUnitAddEdit/tests/index.test.tsx

```diff
@@ -0,0 +1,256 @@
+import flushPromises from 'flush-promises';
+import { mount } from 'enzyme';
+import React from 'react';
+import { history } from '@onaio/connected-reducer-registry';
+import { Provider } from 'react-redux';
+import { MemoryRouter, Route, Router } from 'react-router';
+import fetch from 'jest-fetch-mock';
+import { store } from '@opensrp/store';
+
+import { id, locationUnitgroup, sampleHierarchiesList, sampleLocationUnit } from './fixtures';
+import LocationUnitAddEdit from '..';
+import { act } from 'react-dom/test-utils';
+import { sampleHierarchy } from '../../LocationUnitView/tests/fixtures';
+
+describe('location-management/src/components/LocationUnitAddEdit', () => {
+  it('renders without crashing', async () => {
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <LocationUnitAddEdit />
+        </Router>
+      </Provider>
+    );
+    expect(wrapper.props()).toMatchSnapshot();
```

Comment:
We do not need a snapshot here, a shallow seems to be enough [Ref](https://github.com/OpenSRP/web/blob/master/packages/keycloak-user-management/src/components/CreateEditUser/tests/index.test.tsx#L71)

Reply:
updated tests